quickfixj-core/src/main/doc/usermanual/usage/configuration.html

```diff
@@ -1031,39 +1031,69 @@ <H3>QuickFIX Settings</H3>
     <TD>Any nonempty string.</TD>
     <TD>"" (empty string)</TD>
   </TR>
-
   <TR ALIGN="left" VALIGN="middle">
     <TD><I>JdbcMaxActiveConnection</I></TD>
     <TD>Specifies the maximum number of connections to the database.</TD>
-    <TD>Any number</TD>
+    <TD>Positive number</TD>
     <TD>32</TD>
   </TR>
   <TR ALIGN="left" VALIGN="middle">
-    <TD><I>JdbcMaxActiveTime</I></TD>
-    <TD>Specifies if the housekeeper comes across a thread that has been active for longer than
-      this (milliseconds) then it will kill it. So make sure you set this to a number bigger than your
-      slowest expected response!</TD>
-    <TD>Any number</TD>
-    <TD>5000</TD>
+      <TD><I>JdbcMinIdleConnection</I></TD>
+      <TD>Controls the minimum number of idle connections that HikariCP tries to maintain in
+          the pool, including both idle and in-use connections. If the idle connections dip
+          below this value, HikariCP will make the best effort to restore them quickly and
+          efficiently.
+      </TD>
+      <TD>[0, JdbcMaxActiveConnection]</TD>
+      <TD>Same as JdbcMaxActiveConnection</TD>
   </TR>
   <TR ALIGN="left" VALIGN="middle">
     <TD><I>JdbcMaxConnectionLifeTime</I></TD>
     <TD>Specifies the maximum amount of time that a connection exists for before
       it is killed (milliseconds).</TD>
-    <TD>Any number</TD>
-    <TD>28800000</TD>
-  </TR>
-  <TR ALIGN="left" VALIGN="middle">
-    <TD><I>JdbcSimultaneousBuildThrottle</I></TD>
-    <TD>Specifies the maximum number of connections we can be building at any one time.
-        That is, the number of new connections that have been requested but aren't yet
-        available for use. Because connections can be built using more than one thread
-        (for instance, when they are built on demand) and it takes a finite time between
-        deciding to build the connection and it becoming available we need some way of
-        ensuring that a lot of threads don't all decide to build a connection at once.
-        (We could solve this in a smarter way - and indeed we will one day)</TD>
-    <TD>Any number</TD>
-    <TD>32</TD>
+    <TD>Positive</TD>
+    <TD>28800000 ms (8 hours)</TD>
+  </TR>
+  <TR ALIGN="left" VALIGN="middle">
+    <TD><I>JdbcConnectionTimeout</I></TD>
+    <TD>Set the maximum number of milliseconds that a client will wait for a connection from the
+        pool. If this time is exceeded without a connection becoming available, a SQLException
```

Comment:
Is this correct?

Reply:
I was asking myself the same thing since I found "a" as well as "an". Probably depends on how you speak SQL. When spoken as single letters then "an" should be correct. When you speak it more as "seequel" then probably "a" is correct.

Reply:
https://learnsql.com/blog/sql-or-sequel/ :)

Reply:
Good to know. IDEA didn't highlight it so I didn't pay attention.